test/mason/publish-tests/streamline/streamline.chpl

```diff
@@ -0,0 +1,7 @@
+use MasonPublish;
```

Comment:
How about we call this test `create-registry.chpl` and put it in the `create-registry` directory?

While we're at it, I'd be happy for you to rename `publish-tests` to `publish` in this PR to get us 1 step closer to completing #15683 :)